community/README.md

```diff
@@ -3,3 +3,8 @@
 Welcome to the Argo community!
 
 Argo is an open, community driven project to make it easy to use Kubernetes for getting useful work done. [This document](https://github.com/argoproj/argoproj/blob/main/community/README.md) describes the organizational structure of the Argo Community including the roles, responsibilities and processes that govern Argo projects and community.
+
+## Argo Workflows Sustainability Effort
+
+Argo Workflows is seeking more community involvement and ultimately more [Reviewers and Approvers](https://github.com/argoproj/argoproj/blob/main/community/membership.md) to help keep it viable. Please see [this document](sustainability_effort.md) for more information.
```

Comment:
```suggestion
Argo Workflows is seeking more community involvement and ultimately more [Reviewers and Approvers](https://github.com/argoproj/argoproj/blob/main/community/membership.md) to help keep it viable.
Please see [Sustainability Effort](sustainability_effort.md) for more information.
```

- Avoid ambiguous "this document", [per k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#links)
- Prefer 1 sentence per line of markdown [per style guide](https://argoproj.github.io/argo-workflows/doc-changes/)

Reply:
Will do. Just curious: I'm not sure if I understand "prefer 1 sentence per line of markdown". I can see that it could sometimes be more readable that way. But do you think it should be followed when there are multiple sentences that flow well together? It does say "prefer", so it's probably not a hard and fast rule.?